otter/integration/tests/test_convergence.py

```diff
@@ -989,6 +990,101 @@ def test_recover_from_identity_auth_failures(self):
             max_entities=10)
         yield self.helper.start_group_and_wait(group, rcs, desired=5)
 
+    @inlineCallbacks
+    def test_error_reasons_are_updated(self):
```

Comment:
How about skipping this with link to #1578 and merging it? Then I can merge #1608 with this enabled. That way we that merge will be synchronous with the test.